src/nrnpython/utils.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
```

Comment:
The file can be called `nrnpython/cast_tuple.hpp`.

Reply:
Done